src/main/java/sirius/kernel/di/transformers/AutoTransformLoadAction.java

```diff
@@ -153,27 +153,32 @@ private T createInstance(Object source)
     }
 
     @Override
-    public void handle(@Nonnull MutableGlobalContext ctx, @Nonnull Class<?> clazz) throws Exception {
+    public void handle(@Nonnull MutableGlobalContext mutableContext, @Nonnull Class<?> clazz) throws Exception {
         if (clazz.isAnnotationPresent(Framework.class)
             && !Sirius.isFrameworkEnabled(clazz.getAnnotation(Framework.class).value())) {
             return;
         }
 
         AutoTransform autoTransform = clazz.getAnnotation(AutoTransform.class);
-        Class<?>[] allTargets = mergeAllTargets(autoTransform);
+        Class<?>[] allTargets = mergeAllTargets(autoTransform.target(), autoTransform.targets());
         for (Class<?> target : allTargets) {
-            ctx.registerPart(new AutoTransformer<>(clazz, autoTransform, target, allTargets), Transformer.class);
+            mutableContext.registerPart(new AutoTransformer<>(clazz,
+                                                              autoTransform.source(),
+                                                              target,
+                                                              allTargets,
+                                                              autoTransform.priority()), Transformer.class);
         }
     }
 
-    private Class<?>[] mergeAllTargets(AutoTransform autoTransform) {
-        Class<?> target = autoTransform.target();
-        if (!target.equals(Object.class)) {
-            Class<?>[] targets = autoTransform.targets();
-            Class<?>[] mergedTargets = Arrays.copyOf(targets, targets.length + 1);
-            mergedTargets[targets.length] = target;
-            return mergedTargets;
+    protected Class<?>[] mergeAllTargets(Class<?> target, Class<?>[] targets) {
+        if (targets.length == 0) {
+            return new Class<?>[]{target};
+        }
```

Comment:
Why do we need this check? Wouln't this bring the possibility to create a transformer to Object?